<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="CustomerService_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1031</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Home">
     <attribute name="title">
      <string>Home</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Robot">
     <attribute name="title">
      <string>Robot</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1011</width>
        <height>581</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Description</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>11</y>
          <width>331</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>10</y>
          <width>311</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView_3">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>10</y>
          <width>331</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Add to Cart</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>290</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Add to Cart</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>300</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Add to Cart</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>300</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>300</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_3">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>300</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Shopping Cart</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Checkout</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Admin">
     <attribute name="title">
      <string>Admin</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="CustomerService">
     <attribute name="title">
      <string>Customer Service</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>291</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ASK AN EXPERT!

Methods to get service

-&gt; By Phone
 (123) - 456 - 7891

-&gt; By email

-&gt;Visit a store



</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>651</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SimSun</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Get in touch with us </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="messagebox">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>400</y>
        <width>231</width>
        <height>70</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sendButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>490</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="labelFirstName_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>190</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="labelLastName_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>260</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="labelCompanyName_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelFirstName">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>FIRST NAME</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLastName">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>LAST NAME</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCompanyName">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>290</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>COMPANY NAME</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMessage">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>360</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>MESSAGE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Review">
     <attribute name="title">
      <string>Review</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="FAQ">
     <attribute name="title">
      <string>FAQ</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Help">
     <attribute name="title">
      <string>Help</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClickHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>715</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClickHandler()</slot>
 </slots>
</ui>
